libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -240,20 +240,23 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     return availableLocales[locale] || this.includedLocales.enGB; // Default to enGB if injected locale doesnt exist
   }
 
+  private formatDateLabel(): string {
+    const localeDateFormats = {
+      da: 'd. MMMM',
+      'en-GB': 'd MMMM',
+      'en-US': 'MMMM d',
+    };
+
+    const dateFormat = localeDateFormats[this.locale.code] || 'MMMM d';
```

Comment:
Consider making it explicit that we default to `en-US`:
```suggestion
    const defaultDateFormat = localeDateFormats['en-US'];
    const dateFormat = localeDateFormats[this.locale.code] || defaultDateFormat;
```